resources/js/src/app/shared/interfaces.ts

```diff
@@ -32,9 +32,14 @@ export interface Facet {
 }
 
 export interface FacetValue {
+    count?: string;
+    image?: string;
     id?: string;
     name: string;
     imageUrl?: string;
+    items?: [];
```

Comment:
By declaring `[]` as a type, you implicitly say that this is an array that will always be empty. What is even the purpose of this?